src/toolchain-conventions.adoc

```diff
@@ -216,6 +216,43 @@ As of November 2021 the additional tail-call entry points are only
 implemented in compiler-rt, and calls will only be generated by LLVM
 when the option `-mllvm -save-restore-tailcall` is specified.
 
+== Profile-based format
+
+Profiles names will be a valid list of released profiles, it should be recognized and used
+in the `-march=` option. The benefit of using the `-march` option is easy for toolchain
+parsing the profiles string and expanding it into normal extensions combinations. 
+
+Profiles format has the following BNF form `"-march="<profile-name>"_"[option-ext]*`.
+
+`profile-family-name ::= "i" | "m" | "a" | "b"`
+
+`profile-ratified-year-version ::= "20" | "22" | "23"`
+
+`privilege-mode ::= "u" | "s" | "m"`
+
+`isa-xlen ::= "64" | "32"`
+
+`profile-name ::= "rv"<profile-family-name><profile-ratified-year-version><privilege-mode><isa-xlen>`
```

Comment:
I would suggest just say it should use lowercase of profile name instead of list the naming rule here, because: 1) That may change...like `b` was not defined when RVA20 and RVA22 profile, 2) that may derived some profile not really support, e.g. rvb20u32 is valid according the rule but it's not existing 

```suggestion
`profile-name ::= 'profile-name-in-lowercase'`
```